projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -1069,34 +1069,23 @@ function get_paywall_blocks( $newsletter_access_level ) {
 		// translators: %s is the name of the site.
 		: esc_html__( 'Subscribe to get access to the rest of this post and other subscriber-only content.', 'jetpack' );
 
+	$button_text     = esc_html__( 'Log in', 'jetpack' );
+	$sign_in_link    = 'https://subscribe.wordpress.com/memberships/jwt?site_id=' . \Jetpack_Options::get_option( 'id' ) . '&redirect_url=' . get_current_url() . '&v2';
```

Comment:
I see the v2 now

Reply:
> I don't get the
if ( isset( $_GET['v2'] ) ) {

We need to distinguish if the jetpack site using `subscribe.wordpress.com/memberships/jwt` has the `wp-json/jetpack/v4/subscribers/auth` endpoint